### DISCUSSION TITLE
作為學生的賺錢方式

### DISCUSSION THREAD

**Discussion by Yanxxx7979** [1101-GS4719-C/Python-Community-GS4719-C#1791] [General]
最近突然覺得自己好像已經是個大人了(不是很想長大:))
開始關心起一些理財的問題
自己本身有接家教，每個月會有一些固定收入
也聽說過有朋友
1.去學校附近餐廳打工(我也有打過，坦白講學校附近餐廳用餐時間都超爆忙但又不長，所以能賺的時間也不多加上最低時薪個人覺得相當不ㄏㄜˊ)
2.買股票(來錢快的，聽說有人玩當沖，對我來講很可怕)
還有甚麼轉錢或是理財方式嗎?歡迎大家分享!

**Comment by 1chooo** [comment]
我覺得還是趁大學累積自己吧！現在再怎麼轉大多人都是靠工時堆起來的，還不如畢業後轉一波大的，所以我覺得軟實力跟軟實力比較重要！

**Comment by 111426027** [comment]
除非真的有什麼特殊才能，不然學生沒辦法賺到很多錢，趁學生時期多精進自己的能力，以後出社會才可能有更高的薪資。

**Comment by davidteng00000** [comment]
以我而言，我有把我的資產拿去做基金的投資，資金的好處是在獲利的同時也能保持較低的風險。同時，我也有在實習--負責網頁的維護並且開發一些系統，這些是我主要的賺錢方式

**Comment by Ryanlolry** [comment]
我認為在學生時期處分真的缺學費或生活費，要不然我不會選擇去打工，畢竟用打工時間多精進自己的學業能力，感覺比較有價值。

**Comment by uwje** [comment]
我認為在學生時期還是好好讀書就好，除非無法有充足的學費或生活費，否則花時間在讀書上在未來能有更多的收穫，畢竟現在打工頂多領基本時薪，如果好好讀書未來能領更多。

**Comment by Tomken123** [comment]
可以開直播或是拍一些youtube的影片

**Comment by mrweixd** [comment]
大人會擔心錢，小孩會擔心課業，大學生會擔心兩者

**Comment by thz0000** [comment]
我覺得學生時期唯一值得做的打工就是家教，如果沒有經濟壓力的話我也不推薦接過多的工作。
以我而言，我最多在寒暑假會去接家教，體驗一下順便賺點外快。
但以長遠來看，現在認真讀書、學習技能的價值是最高的。

**Comment by TobywithDino** [comment]
我覺得做家教的CP值超級高，因為學生其實沒什麼時間可以打工，如果只是在一般的餐廳parttime的話，通常都拿最低時薪。
所以家教就很賺，而且還可以複習國高中知識... 有沒有用我不確定

**Comment by chiuwangwang** [comment]
我認為還是要懂一些理財的規劃，開源節流，我上高中就有開始每天記帳，這樣每月底就可以檢討哪些是真的不必要。現在也有買些股票，雖然我感覺不會賺到錢，但我就放個10年20年，不領出來就不算認輸。

**Comment by YifenLin** [comment]
我自己之前在decard上有看到有人分享
"適合大學生的五種賺錢方法"
https://www.dcard.tw/f/persona_lazybear/p/242028822
1⃣️經營網拍
2⃣️經營自媒體
3⃣️當家教
4⃣️問卷/訪談賺錢
5⃣️賺錢APP
我自己認為3, 4, 5點比較可以實行，不過我自己的觀點是身為大學生還是把時間花在投資自己，多學一點技能，錢之後再賺就好了。

**Comment by SusuTima** [comment]
我覺得如果那個打工性質是可以增加自身經驗的，那就可以花時間去嘗試，畢竟多看看增加眼界也好，但如果在不缺錢的情況下，瘋狂打工賺錢，感覺會有失身為一位大學生應該學習並增進自身能力的本質。

**Comment by KH1126** [comment]
個人目前有在投資股票。
有分長期和短期來購買股票。
我之前以短期購買的股票是聯電，陽明，有小賺一筆。長期則是投資較看好長期發展的公司，如台積電。
然而台積電最近股票大漲，在想說要不要賣掉了。

**Comment by lyykey** [comment]
我目前除了家裡資助的生活費，並沒有額外固定的收入。
但是身為學生，我盡可能的善用學校能提供的資源，像是書卷獎的獎學金(雖然超少)，再來就是優秀獎學金，以及朱順一、羅家倫的獎學金，還有做學校教授 RA，主要的不固定收入都是來自學校給予的錢。
希望大家能夠多利用學校的資源，也歡迎推薦給我~

**Comment by sunset513** [comment]
我聽身邊的同學們分享大多有以下幾種方式:

校內工讀機會:
像是經常有一些校內的工作機會,如服務學習中心、系辦等，優點是地點就在校園內方便，我自己也有在做，我個人蠻推薦的。

餐飲服務業:
後門或是宵夜街附近，可以學到一些服務經驗，但我沒有試過，沒辦法分享太多。

課輔班或家教:
好像是賺最多錢的，也可複習並鞏固自己的知識，我的朋友們都很愛接，一直接一直爽。

**Comment by woohsin** [comment]
原本小時候的夢想是去餐廳端盤子，結果第一份收入是家教，就算是家教裡比較低的薪水也幾乎是基本時薪兩倍，所以我已經無法接受其他打工了，但是家教其實很難找，除非有人讓渡給你，不然就是上臉書社團瘋狂發文。

**Comment by benbebobo149** [comment]
在學校裡，確實存在許多賺錢的機會，有時候會有些奇奇怪怪的工讀，例如文書處理等，而且還可以在家裡進行。但這些工作可能不會直接被標示為文書處理，而是採用一些更具吸引力的名稱，像是標記繪圖工作等等。起初，這些工作看起來可能非常繁雜，費時費力，但當你學會使用程式和工具時，就能夠派上用場了。例如，對於十小時的工作，如果你花兩小時寫程式，剩下的時間就都是你實際賺到的。

然而，在賺錢的同時，我認為也應該思考一下自己真正想要的是什麼。是追求快速的財富積累，還是追求事業上的成長和發展？有些工作可能在開始時收入較低，但隨著時間的推移，卻能帶來更大的成長機會。因此，我建議你不要被眼前的利益所蒙蔽，可以考慮嘗試一些實習或接案等機會，這些工作可能會為你帶來更多長遠的收益和滿足感。感謝同學的分享，讓我有機會思考這些重要的議題。

**Comment by jeff110303549** [comment]
我覺得若是家裡情況允許，在大學期間應該還有很多除了賺錢以外更重要的事情，畢竟有些是大學不做可能一輩子也不會去做了；不過我倒是認同不論任何科系，大學生應該去學習理財相關的知識或是對某些領域的發展，想法參考自股神巴菲特的「棒球理論」，有興趣歡迎去youtube 上看看。
連結：https://youtu.be/BAMiemNe2fk?si=lnIppU3XpSqAQEQo

**Comment by YANRONG0123** [comment]
由於家裡給的資源相較足夠，所以我也不太想去打工，反而是精進自己本科系以外的能力，可能還比較重要，至於賺錢的方法，也可以拚一下學業優異的獎學金，不僅分數好看之後用得上，獎學金累積起來也是一筆可觀的收入。

**Comment by 20050201Shawn** [comment]
我有在網路上畫一些圖賺一些小錢，但因為我畫畫很慢所以換算成時薪通常是50塊以下，雖然一部分是出於好玩才畫的。認同樓上說大學先好好充實自己，趁還沒要出社會享受當爸寶媽寶的日子(沒

**Comment by tzurou** [comment]
我自己剛上大學時也是想著可以打工賺一點娛樂費，但是後來發現課業壓力太大就放棄了，現在大三在學校行政工讀，時間可以自己選，不會占用到太多個人時間還可以讀書，我覺得如果不缺錢的話可以考慮，但如果想賺大錢的話我覺得還是要先花時間做研究，現在充實自己的時間和以後賺的錢比絕對划算！

**Comment by happy246810** [comment]
我覺得賺錢這件事必須建立在保持學業成績上，畢竟在大學本身就是一間學校，打工賺錢只是一個附加的選項，畢竟打工的錢跟你出社會賺的錢比起來只是一小部分。而讀書讀得好，也有機會拿到書卷獎，所以我的結論是不需要特意去打工，除非真的有需要或想賺娛樂費。

**Comment by LinChloe** [comment]
雖然大家都說打工很沒收穫又浪費時間，
但我覺得打不同的工能讓自己開拓眼界，雖然這份工很爛，但你知道爛在哪裡，
之前打了一份工雖然累，但見識到很多的人，也讓我珍惜目前的環境。

**Comment by yxin214** [comment]
我覺得接家教是很棒得的賺錢方式！尤其是教和自己科系相關的科目，在備課過程可以幫助自己釐清一些觀念(覺得自己讀得好和要把別人教得好完全是兩個不同的層次...)，而且家教學生有進步時也會很有成就感。

**Comment by thomasxshen** [comment]
我覺得家教很棒也很好，想比起條碼師來說來錢快也省時間，畢竟大學生的時間實在太過零碎，其實外面找的工作很難穩定，也很難安排，像我常常就是因為卡到自己的時間而沒辦法做該工作只好放棄，家教一周可能也就二到四小時而已，也不須付出甚麼勞動，其實真的是一筆不錯的收入。

**Comment by William08290829** [comment]
我認為現在開始學賺錢是一個重要的生活技能。雖然學業是首要的，但掌握賺錢的方法和技巧也是很有價值的。這不僅有助於經濟獨立，還可以培養創業精神、人際關係和時間管理能力

**Comment by Eileen050824** [comment]
我覺得打工雖是獨立且為自己負責的開始，但也不要印因為想要多點收入拼命地打工而荒廢學業。我自己學期中是沒有在打工，但我平常花費也不多(盡量幫家裡減少開銷)，寒暑假靠寫一些作品+幫以前補習班的忙多一份收入，加上儲蓄和適當投資，基本上我覺得自己夠用就好！

**Comment by willychiuchiu** [comment]
自己因為家裡沒有給生活費，但又不想在學期中花時間打工，所以通常利用寒暑假努力打工，能存多少儘量存，然後再利用從小存的小小投資股票，自己慢慢研究，雖然本金不大賺不了太多，但是不無小補！

**Comment by Hahahahbla** [comment]
除了家教收入外，考慮打工、買股票等。有人透過實習賺取收入。但自己認為專注學業，提升軟實力，機會更大為上策。

**Comment by YuanWang08** [comment]
可以趁現在大學錢還不多All in 加密貨幣，確保基本的生活品質就好，運氣好畢業後直接財富自由，就算輸了也不會到太慘，反正現在大學也沒多少錢可以輸，不如趁年輕的時候拼一波！

**Comment by buoioiry** [comment]
身邊蠻多認識的都在做家教，感覺對大學生來說是不錯的工作，可以賺一些多餘的生活費或是減輕家裡負擔，如果只接一兩個也不致於影響學業

**Comment by yyujimmy** [comment]
理財是一個很重要的議題，特別是當你開始關心自己的財務狀況時。除了家教和打工外，還有一些其他的理財方式可以考慮：

存款和定期存款：將一部分收入存入銀行作為儲蓄，可以確保有一定的備用資金，並且可以通過定期存款來獲得更高的利息。
基金投資：可以考慮通過投資基金來分散風險，比如指數基金或股票基金。相對於直接買賣股票，基金投資通常更穩健，並且能夠參與股市的成長。
房地產投資：如果有足夠的資金，可以考慮投資房地產，例如購買房屋出租或投資房地產基金。房地產通常是一個穩定的投資選擇，並且可以獲得穩定的租金收入。
教育基金：如果有計劃進修或學習更多技能，可以考慮將一部分收入用於建立教育基金，以支付未來的學費和相關費用。
自我提升：除了投資外，也可以考慮花費一部分時間和金錢在自我提升上，例如參加課程、培訓或是購買書籍。這些投資可以幫助你提升自己的技能和知識，從而提高未來的收入和生活品質。
最重要的是，選擇適合自己的理財方式，並且根據自己的目標和風險承受能力來進行投資和規劃。建議在做出任何重大投資決策之前，先咨詢專業的理財顧問或者進行充分的研究和分析。

**Comment by wanka2030** [comment]
經營自媒體，以經營IG、Dcard當副業，而自媒體賺錢的方法，除了有接收影片紅利、也有廠商會請他寫業配文。營自媒體非常考驗抗壓性及自律，可能會因為網上的批評影響心情，同時經營自媒體不像家教、賣場會有固定的時間，很容易變成三天捕魚兩天曬網，有一點時間管理的能力同時生活比較不忙的人可能會比較適合。

**Comment by jeff110303549** [comment]
理財是一個很重要的議題，尤其是隨著人生階段的轉變，像是進入大學或是工作後。除了你提到的家教和打工，還有一些其他的理財方式可以考慮：

1. 儲蓄：建立緊急儲蓄基金，以應對突發狀況或未來的支出。

2. 投資基金：考慮投資於股票、債券或指數基金等投資工具，可以分散風險並實現長期增長。

3. 存款：定期存款或定存是一種相對安全的投資方式，可以享有一定的利息收益。

4. 理財規劃：可以諮詢理財顧問或金融專家，制定個人化的理財計劃，以達到長期財務目標。

5. 學習理財知識：不斷學習關於理財的知識和技能，可以幫助你做出更好的財務決策，提高財務管理能力。

重要的是選擇適合自己的方式並保持理性，不要因為別人的成功或失敗而盲目跟隨。

**Comment by asd99263fg** [comment]
大學生賺錢的方式有很多種，包括兼職工作如在咖啡店、餐廳或零售店工作，這些工作時間靈活，適合學生安排學業之外的時間。此外，學生可以利用自己的技能和專長進行家教、寫作或設計等自由職業工作。參與校內的助教、研究助理或行政助理等職位，不僅能賺取收入，還能增加專業經驗。隨著數位時代的發展，學生也可以通過線上平台進行電商、創作內容或接收線上任務來賺錢。

**Comment by Tchang323** [comment]
當家教? 只不過在中央交通超級不方便，如果沒機車，要到山下上課真的蠻不方便的

**Comment by Tchang323** [comment]
可以去餐廳大打工、學校行政單位、系辦、活動工作人員、當家教
其實還蠻多賺錢的機會
